core/commands/partials/terraform_configuration_generator.rb

```diff
@@ -138,7 +138,7 @@ def make_node_params(node, box_params)
   def need_attached_disk?(node)
     ConfigurationGenerator.parse_products_info(node).map do |product|
       product['name']
-    end.include?('clustrix')
+    end.include?('clustrix') || node[1]['attach_disk'] == 'true'
```

Comment:
This meta-information may be put into the corresponding product description. This is not priority direction.